Bio/PDB/PSEA.py

```diff
@@ -30,12 +31,22 @@ def run_psea(fname):
     Note that P-SEA will write an output file in the current directory using
     the input filename with extension ".sea".
 
-    Note that P-SEA will write output to the terminal while run.
+    Note that P-SEA will not write output to the terminal while run unless
+     verbose is set to True.
     """
-    subprocess.call(["psea", fname])
     last = fname.split("/")[-1]
     base = last.split(".")[0]
-    return base + ".sea"
+    cmd = ["psea", fname]
+
+    p = subprocess.run(cmd, capture_output=True)
```

Comment:
You can get around using these `decode` calls by using `universal_newlines=True` or `text=True` (the latter is an alias for the format introduced for clarity but is only avaiable in Py 3.7 and onwards). Then, `p.stdout` and `p.stderr` will not be byte strings anymore.

Reply:
Very nice, I should have looked at the docs more! I've used the `universal_newlines` on the off chance that someone is trying to use <3.7, though I know the docs say that it isn't supported.